perf/perf_pmu.py

```diff
@@ -209,6 +227,40 @@ def test_cpu_event_count(self):
         # cpu events count
         self._check_count('cpu')
 
+    def test_sysfs_events(self):
+        devices_events = ['cpu',  'hv_24x7', 'hv_gpci']
+        for type_events in devices_events:
+            directory_base = '/sys/bus/event_source/devices'
+            directory = os.path.join(directory_base, type_events, 'events')
+            print(directory)
+            random_files = self.get_random_filenames(directory)
+            self._create_temp_user()
+            if not process.system('id test_pmu', sudo=True, ignore_status=True):
+                for file in random_files:
+                    eventdir = os.path.join(directory, file)
+                    result = process.run("su - test_pmu -c 'echo 1 >  %s'" % eventdir,
+                                         shell=True, ignore_status=True)
+                    output = result.stdout.decode() + result.stderr.decode()
+                    self.log.info("Test passes for %s of %s" %
+                                  (file, type_events))
+                if 'Permission denied' not in output:
+                    self.fail("Able to read  %s of %s as normal user" %
+                              (file, type_events))
+                self._remove_temp_user()
```

Comment:
can you move user creation and removal outside loop, currently its happening 3 times for the same test